Resources/Prototypes/Catalog/VendingMachines/Inventories/vendomat.yml

```diff
@@ -10,3 +10,5 @@
     MatterBinStockPart: 4
     CapacitorStockPart: 4
     MicroManipulatorStockPart: 4
+  contrabandInventory:
+    KitchenKnife: 1
```

Comment:
Scalpel would be funnier

Reply:
I'm not sure why it'd be funnier, but it serves the exact same purpose. And without looking it up, the damage difference is what, 7 to 8?

I kind of like this one, and it feels like a more relevant fit to the VendoMat, it feels more "techy" than the kitchen knife does. I'll likely just do that swap without further prompting.

Reply:
Scalpel is fast weapon so its actually 12 dps.

Reply:
Fair point. It's not amazing, I've had to defend myself as a Chemist using one on several occasions at this point, but that's good to know.

I still like this suggestion. I mostly just want to throw a "legitimate" cutting tool to the resourceful who can get their hands on a screwdriver and wirecutter, and are willing to risk getting shocked finding the manager wire that round.

Reply:
Kitchen Knife swapped for Scalpel.